openedx/core/djangoapps/api_admin/forms.py

```diff
@@ -0,0 +1,42 @@
+"""Forms for API management."""
+from django import forms
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.api_admin.models import ApiAccessRequest
+
+
+class ApiAccessRequestForm(forms.ModelForm):
+    """Form to request API access."""
+
+    company_name = forms.CharField(label=_('The name of your company.'))
+    company_address = forms.CharField(label=_('The contact address of your company.'))
```

Comment:
@pbaruah do we actually need a mailing address? What are we going to do with it?